operations/validate_csm_health.md

```diff
@@ -61,335 +54,100 @@ Health Check scripts can be run:
-
-<a name="net-extdns"></a>
-### 1.5 Verify ability to resolve external DNS
-
-If unbound is configured to resolve outside hostnames, then the following check should be performed. If unbound is not configured to resolve outside hostnames, then this check may be skipped.
-
-Run the following on one of the master or worker nodes (not the PIT node):
-
-```bash
-ncn# nslookup cray.com ; echo "Exit code is $?"
-```
-
-Expected output looks similar to the following:
-```
-Server:         10.92.100.225
-Address:        10.92.100.225#53
-
-Non-authoritative answer:
-Name:   cray.com
-Address: 52.36.131.229
-
-Exit code is 0
-```
-
-Verify that the command has exit code 0, reports no errors, and resolves the address.
-
-<a name="net-spire"></a>
-### 1.6 Verify Spire Agent is Running on Kubernetes NCNs
-
-Execute the following command on all Kubernetes NCNs (all worker nodes and master nodes), excluding the PIT node (booted from the LiveCD):
+And the Kubernetes test suite via:
 
 ```bash
-ncn# goss -g /opt/cray/tests/install/ncn/tests/goss-spire-agent-service-running.yaml validate
-```
+ncn# /opt/cray/tests/install/ncn/automated/ncn-kubernetes-checks
+```
+Review the output for `Result: FAIL` and follow the instructions provided to resolve any such test failures. With the exception of the [Known Test Issues](#autogoss-issues), all health checks are expected to pass.
+
+The ncn-healthcheck suite covers the following (as appropriate for worker, master and storage nodes):
+* Commands Available
+* Interfaces Has IP
+* Bond Members Have Links
+* Interfaces Has VLAN
+* Bond Members Have Correct MTU
+* Nodes DNS Resolvable
+* Unbound Nameserver In resolv.conf
+* Sealed Secrets Key Exists
+* Kubernetes StorageClasses
+* Kubernetes Velero Backup Storage Locations Ready
+* Kubernetes Velero Daily Vault Backup Schedule Exists
+* Kubernetes Velero No Failed Backups
+* Spire Health Check
+* Kubernetes Query BSS Cloud-init for spire meta data
+* Spire Pods
+* Verify spire-agent is enabled and running
+* Validate token key ID exists in spire jwks
+* Verify spire api fetch of jwt
+* Service 'etcd' Running
+* Etcd ca.crt file exists
+* Etcd client crt file exists
+* Etcd client key file exists
+* Validate the Kubernetes etcd on this node is healthy
+* Default Gateway Same as CAN Gateway
+* Verify cray etcd is healthy
+* Validate that the local platform CA bundle matches the one in cloud-init
+* Check clock skew on k8s and storage nodes
+* Validate PodSecurityPolicy is enabled
+* Resolve external DNS
+* Kubernetes Check cray-vault Vault Cluster Status
+* Kea has Active DHCP Leases
+* All Kubernetes Etcd Clusters have a Backup Created within the Last 24 Hours
+* All Etcd Pods in a Cluster are Running on a Different Worker Node
+* Etcd Database Health
+* Etcd Cluster Endpoint Health
+* Validate the kubernetes etcd cluster has three active members
+* Service 'etcd' Running
+* Validate the Kubernetes etcd on this node is healthy
+* Switch BGP Neighbors
```

Comment:
I don't see how this list is useful, many of them aren't going to make sense to an admin, like this last one.

Reply:
OK I can remove - I was trying to give a sense to what is in each suite so I took the Title from each test.  But I agree, might be best to just not include this.  Removing ...